Client/Coordinators/PasswordManagerCoordinator.swift

```diff
@@ -71,18 +71,32 @@ class PasswordManagerCoordinator: BaseCoordinator,
     func pressedPasswordDetail(model: PasswordDetailViewControllerModel) {
         let viewController = PasswordDetailViewController(viewModel: model)
         viewController.coordinator = self
+        sendLoginsManagementLoginsTappedTelemetry()
         router.push(viewController)
     }
 
     func pressedAddPassword(completion: @escaping (LoginEntry) -> Void) {
         let viewController = AddCredentialViewController(didSaveAction: completion)
         let navigationController = UINavigationController(rootViewController: viewController)
         navigationController.modalPresentationStyle = .formSheet
+        sendLoginsManagementAddTappedTelemetry()
         passwordManager?.present(navigationController, animated: true)
     }
 
     func openURL(url: URL) {
         parentCoordinator?.settingsOpenURLInNewTab(url)
         parentCoordinator?.didFinishPasswordManager(from: self)
     }
+
+    private func sendLoginsManagementAddTappedTelemetry() {
```

Comment:
I rather not have coordinators handle Telemetry. I think this should be handled by the VC